### PR TITLE
chore: add new deployment to subgraph

### PR DESCRIPTION
**Pull request by tmigone**

### CODE REVIEW

File: subgraph/subgraph.yaml [hunk 1]
```diff
@@ -32,3 +32,33 @@ dataSources:
         - event: AuthorizedSignerRemoved(indexed address,indexed address)
           handler: handleAuthorizedSignerRemoved
       file: ./src/subscriptions.ts
+  - kind: ethereum
+    name: Subscriptions-native
+    network: arbitrum-one
+    source:
+      address: '0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512'
```
[the remark this answers is not in the document]
Comment (tmigone) [reply subgraph/subgraph.yaml]: It doesn't 😄 , I fixed it by using mustache templates and generating different manifests for different networks

[the remark this answers is not in the document]
Comment (hasparus) [reply subgraph/subgraph.yaml]: nice solution, but oh dude how I detest YAML xD

**Review by Theodus — APPROVED** [review]
LGTM

File: subgraph/subgraph.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (tmigone) [reply subgraph/subgraph.yaml]: Yes i hate it as well lol

**Pull request merged**